user_events.py

```diff
@@ -205,9 +205,17 @@ def user_an_event(id):
                     subEvent['isPublished'] = True
                 else:
                     subEvent['isPublished'] = False
-            else:
+            elif 'eventid' in subEvent:
```

Comment:
@bingzhang, As discussed during today's meeting, it might be better to use the camelcase naming format (`eventId`) like in other places to avoid confusion. I will approve this PR. Please look into this as part of a code cleanup task. Thanks.

Reply:
Thanks. Created issue here: https://github.com/rokwire/events-manager/issues/831